app/graphql/types/query_type.rb

```diff
@@ -1,17 +1,31 @@
 module Types
   class QueryType < Types::BaseObject
+    description "The query root of this schema"
+
     # Add `node(id: ID!) and `nodes(ids: [ID!]!)`
     include GraphQL::Types::Relay::HasNodeField
     include GraphQL::Types::Relay::HasNodesField
 
     # Add root-level fields here.
     # They will be entry points for queries on your schema.
 
-    # TODO: remove me
-    field :test_field, String, null: false,
-                               description: "An example field added by the generator"
-    def test_field
-      "Hello World!"
+    field :daily_stat_by_id, DailyStatType, null: true do
+      description "get daily stats of the user by daily stats id"
+      argument :id, ID, required: true
+    end
+
+    def daily_stat_by_id(id:)
+      DailyStat.find(id)
```

Comment:
how do we plan to protect this to the signed in user? should it be

```
context[:current_user].daily_stats.find_by(id: id)
```

or

```
DailyStat.find_by(id: id, user_id: context[:current_user].id)
```

?

Reply:
great question